lib/mpack-amalgamation-1.0/CMakeLists.txt

```diff
@@ -2,6 +2,10 @@ set(src
   src/mpack/mpack.c
   )
 
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c99")
+if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
+else()
```

Comment:
You need to upstream this changes to https://github.com/ludocode/mpack.
We generally avoid hand-patching upstream libraries.